cfme/services/catalogs/catalog_item.py

```diff
@@ -3,14 +3,17 @@
 from collections import OrderedDict
 from cfme.fixtures import pytest_selenium as sel
 from cfme.web_ui import Form, Radio, Select, Table, accordion, fill,\
-    flash, form_buttons, menu, tabstrip, DHTMLSelect, Input
+    flash, form_buttons, menu, tabstrip, DHTMLSelect, Input, Tree
 from cfme.web_ui import toolbar as tb
 from utils.update import Updateable
 from utils.pretty import Pretty
+from utils.version import current_version
 
 cfg_btn = partial(tb.select, "Configuration")
 accordion_tree = partial(accordion.tree, "Catalog Items")
 policy_btn = partial(tb.select, "Policy")
+dynamic_tree = Tree("//div[@id='basic_info_div']"
+                    "/div/div/div/div/div/div/ul[@class='dynatree-container']")
```

Comment:
Can we get a cleaner locator here? Perhaps `"//div[@id='basic_info_div']//ul[@class='dynatree-container']"`?